### ISSUE TITLE
Installing on Arch

### ISSUE THREAD

**Issue by Quardah** [LucKeyProductions/Quatter#5]
Hi, I ran the install script, this is what i recieved:

```
[jape@T540p Quatter]$ ./install.sh 
sudo: apt-get: command not found
Already on 'master'
Your branch is up-to-date with 'origin/master'.
Already up-to-date.
Usage: cmake_clean.sh /path/to/build-tree
./cmake_generic.sh: line 69: cmake: command not found
make: *** No targets specified and no makefile found.  Stop.
Already up-to-date.
Info: creating stash file /home/jape/My Games/Quatter/Quatter-build/.qmake.stash
g++ -c -pipe -std=c++1y -O2 -march=x86-64 -mtune=generic -O2 -pipe -fstack-protector-strong -Wall -W -fPIC -DDATADIR=\"~/.local/share/quatter\" -I../Quatter -I. -I../Quatter/Urho3D/include -I../Quatter/Urho3D/include/Urho3D/ThirdParty -I/usr/lib/qt/mkspecs/linux-g++ -o luckey.o ../Quatter/luckey.cpp
In file included from ../Quatter/luckey.cpp:19:0:
../Quatter/luckey.h:22:27: fatal error: Urho3D/Urho3D.h: No such file or directory
 #include <Urho3D/Urho3D.h>
                           ^
compilation terminated.
make: *** [Makefile:626: luckey.o] Error 1
chown: cannot access '/home/jape/.local/share/luckey/quatter/': No such file or directory
chown: cannot access '/home/jape/.local/share/icons/quatter.svg': No such file or directory
./install.sh: line 14: update-icon-caches: command not found

```

Should i make any operations before trying again?

Please update the readme.

**Comment by Modanung** [comment]
What distribution are you running?

**Comment by Quardah** [comment]
I am running Arch Linux (Package Manager is pacman)

I checked into the install script, you run the following at first :

```
./.installreq.sh
./.buildurho.sh
```

The .installreq.sh has apt-get command which is not distro-agnostic. It can only work onto the debian/ubuntu based distributions. 

Also when trying to install what's into .installreq.sh, i get the following 👍 

```
[jape@X250 ~]$ sudo pacman -S libx11-dev libxrandr-dev libasound2-dev
[sudo] password for jape: 
error: target not found: libx11-dev
error: target not found: libxrandr-dev
error: target not found: libasound2-dev
```

It is not present in Arch Linux's repository.

Without the "dev" suffix, i get the following : 

```
[jape@X250 ~]$ sudo pacman -S libx11 libxrandr libasound2
error: target not found: libasound2
```

libasound2 exists on Arch Linux as [alsa-lib](http://rosindex.github.io/d/libasound2-dev/)

The following command works and installs all required dependencies for your program : 

```
[jape@X250 ~]$ sudo pacman -S libx11 libxrandr alsa-lib
```

Maybe with this info you can twist the script a little for it to work on different distributions!

**Comment by Modanung** [comment]
Exactly, these errors were to be expected on non-Debian platforms. And now you mention it, I see Quatter doesn't have any installation instructions in it's readme yet.

If (next to the correct platform) you have the knowledge to modify the scripts in a way that makes them work on both platforms, feel free to send in a much appreciated pull request.
I would gladly apply these changes to all other LucKey projects.

And maybe the installation paths in the .pro file would need to be diversified too?

**Comment by Quardah** [comment]
Hi.

There was also cmake missing to install the program properly.

I will work onto an "install_with_pacman.sh" script that'll make your program work on arch and other distributions using the pacman package manager.

Cheers!
JPBD

**Comment by Modanung** [comment]
Awesome! :)

**Comment by Quardah** [comment]
Hey mate check the Pull Requests ;)

**Comment by Modanung** [comment]
Cool!
Fixed by #6
